tensorflow/compiler/mlir/tensorflow/tests/tf-ops.mlir

```diff
@@ -558,10 +558,90 @@ func @testValidConv2D(%arg0: tensor<256x32x32x3xf32>, %arg1: tensor<3x3x3x16xf32
 
 // -----
 
-// CHECK-LABEL: func @testValidDynamicConv2D
-func @testValidDynamicConv2D(%arg0: tensor<*xf32>, %arg1: tensor<*xf32>) -> tensor<?x?x?x?xf32> {
```

Comment:
Could you add a test case for only dynamic dimension on the batch dim to verify the inference result?

Reply:
Thanks for the review. See `testDynamicBatchConv2D`.

Reply:
How about moving these new tests under shape inference test if it is available?

Reply:
For example, https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/mlir/tensorflow/tests/shape_inference.mlir

Reply:
Thanks for pointing out :) Let me try it real quick.

Reply:
Updated 😃